packages/block-library/src/cover/index.php

```diff
@@ -43,9 +43,19 @@ function render_block_core_cover( $attributes, $content ) {
 			update_post_thumbnail_cache();
 		}
 		$current_featured_image = get_the_post_thumbnail_url();
-		$content                = preg_replace(
+
+		$styles = 'background-image:url(' . esc_url( $current_featured_image ) . '); ';
+
+		if ( isset( $attributes['minHeight'] ) ) {
+			$height_unit = $attributes['minHeightUnit'] ?? 'px';
+			$height		 = " min-height:{$attributes['minHeight']}{$height_unit}";
```

Comment:
2 things here: The null-coalescing operator that @Mamaduka mentioned above, and alignment of the `=` signs 👍 

```suggestion
			$height_unit = empty( $attributes['minHeightUnit'] ) ? 'px' : $attributes['minHeightUnit'];
			$height      = " min-height:{$attributes['minHeight']}{$height_unit}";
```